pkg/notify/messages.go

```diff
@@ -18,31 +24,31 @@ type TaskStateUpdate struct {
 	StepsTotal         int
 }
 
-// Message renders a string representation of a TaskStateUpdate
-func (tsu TaskStateUpdate) Message() string {
-	return fmt.Sprintf(
-		"#task #id:%s %s",
-		tsu.PublicID,
-		tsu.Title,
-	)
+// Message returns a task state change struct
+func (m *Message) Message() *Message {
```

Comment:
not sure what's the use of this method anymore ?

Reply:
Fixed an rebased on master upstream.